crates/holochain_sqlite/src/sql/cell/schema.sql

```diff
@@ -92,6 +92,7 @@ CREATE TABLE IF NOT EXISTS DhtOp (
 
     storage_center_loc          INTEGER   NOT NULL,
     authored_timestamp_ms       INTEGER   NOT NULL,
+    private_entry               INTEGER   NULL,  -- BOOLEAN
```

Comment:
is the query faster checking NULL vs false? If so, totally cool with this, otherwise it might be more semantically readable like:
```suggestion
    private_entry               INTEGER   DEFAULT FALSE NOT NULL,  -- BOOLEAN
```